### ISSUE TITLE
Incorrect remove newlines if use filter newlines_to_br

### ISSUE THREAD

**Issue by damiles** [Shopify/liquid#1196]
If you use the filter newline_to_br you are not cleaning correctly the CLFR as is done in strip_newlines

Its recomended to change line https://github.com/Shopify/liquid/blob/e83b1e415990894c9517f94a8c2020ff825da027/lib/liquid/standardfilters.rb#L298

to:

```
input.to_s.gsub(/\r?\n/, '<br />')
```

This will remove strange carriage character ^M

**Comment by pushrax** [comment]
This is a fair point; the `strip_newlines` filter does remove CR so the Liquid codebase has already conceded that "newline" can mean CRLF. Though it's a historical oddity and personally I'd prefer for the world to stop with CRLF in text documents.

**Comment by damiles** [comment]
Yes, i agree with you that the world must stop use CRLF, but rarelly this occour and is better avoid this issues. I spent 2 hours checking why strip_newlines works and newline_to_br don't clean CRLF, and if we can solve it without stoping world developers will not have a new headache ;)
